DansTextBasedAdventure.js

```diff
@@ -0,0 +1,70 @@
+const readline = require("readline-sync");
+const chalk = require("chalk");
+let nameInput = readline.question(chalk.yellow("Enter your name:"));
```

Comment:
consider moving to start of game so that you can reset when restarting. That way I can have a diff name each time.